### PR TITLE
[storybook] add timeout for page.evaluate

### PR DESCRIPTION
**Pull request by SophieDePaula**
[Trello card](https://trello.com/c/QN2p1v0O/1401-cvent-storybook-execution-hanging)

Looking at the `incomplete` logs from the card above, I can see that we need to render `128` stories. 
In the logs I see `128` times `getting data from story` but only `127` times `done getting data from story` and `running takeDomSnapshot(s) for story`.
[All this logic is in `getStoryData`](https://github.com/applitools/eyes.sdk.javascript1/blob/master/packages/eyes-storybook/src/getStoryData.js#L13)
This leads me to believe that we hang on `page.evaluate` and therefore, I added a timeout of 2 minutes to this operation.

### CODE REVIEW

File: pending-changes.yaml [hunk 1]
```diff
@@ -98,6 +98,7 @@ Apollo SDK:
   bug-fix:
     - Fixed broken links to enums implementation in the README.md
     - Fix rendering issues with Salesforce Lightning design system
+    - Add timeout when attempting to get data for a story
```
[the remark this answers is not in the document]
Comment (benbabayoff8) [reply pending-changes.yaml:101]: @SophieDePaula Looks good to me

**Comment by tourdedave** [comment]
How does adding a timeout to this operation help resolve the customer's issue?

**Comment by SophieDePaula** [comment]
@tourdedave it will prevent the test from hanging and we will know better where the issue is coming from.
In this particular part of the code we are trying to run a few commands on the browser to get info about a story, and according to the user this is happening intermediately so this might indicate that a certain story is not loading properly.

**Comment by tourdedave** [comment]
Okay. Do all tests pass w/ the change (including `yarn eyes-storybook:heavy`)?

**Comment by SophieDePaula** [comment]
@tourdedave I'll run all the tests now

**Pull request merged**